cloud_GCP/terraform/module/README.md

```diff
@@ -86,9 +86,9 @@ module "kentik_gcp_integration" {
 | topic_prefix | Pub/Sub topic prefix and subscription prefix | `string` | `kentik_topic` | no |
 | sink_prefix | Prefix to use with logs sink | `string` | `kentik_pubsub_sink` | no |
 | project | GCP project name | `string` | | yes |
-| name | Exported cloud name in Kentik Portal | `string` | `terraform_aws_exported_cloud` | no |
+| name | Cloudexport entry name in Kentik | `string` | `terraform_aws_exported_cloud` | no |
```

Comment:
`terraform_aws_exported_cloud` is a sub-optimal name for export from GCP

Reply:
Sub-optimal is a nice euphemism ;) Fixed.